pkg/controller/storageclusterinitialization/storageclusterinitialization_controller.go

```diff
@@ -317,11 +342,9 @@ func (r *ReconcileStorageClusterInitialization) ensureStorageClasses(initialData
 			if err != nil {
 				return err
 			}
-		default:
-			return err
 		}
 	}
-	return nil
+	return err
```

Comment:
The unit tests are most probably failing due to these changes you did. Please check this again, and ensure this still behaves as expected by the unit tests. Or fix the unit tests if you think they expect the wrong thing.

Reply:
yeah fixed